index.js

```diff
@@ -25,20 +28,86 @@ app.get('/:name/:days/:record', (req, res) => {
     })
 })
 
+app.get('/:name/:days/:record', (req, res) => {
+    const {name, days, record} = req.params
+
+    res.render('internal', {
+        name,
+        days,
+        record
+    })
+})
+
 app.get('/i/:name/:days/:record', async (req, res) => {
+    renderLastTilt(res, req.params.name, req.params.days, req.params.record);
+})
+
+app.get('/automated/source/:name/tiltou', (req, res) => {
+    const {name} = req.params
+    const lastTilt = new Date().getTime();
+    
+    Tiltados.findOne({name}).then(savedTiltado => {
+        let record = 0;
+        if (savedTiltado) {
+            const daysDiff = daysDifference(lastTilt, savedTiltado.lastTilt)
+            if (daysDiff > record) {
+                savedTiltado.record = daysDiff;
+            }
+            savedTiltado.lastTilt = lastTilt;
+            new Tiltados(savedTiltado).save();
+            renderLastTilt(res, savedTiltado.name, daysDiff, savedTiltado.record < daysDiff ? daysDiff : savedTiltado.record);
+        } else {
+            new Tiltados({name, lastTilt, record}).save();
+            renderLastTilt(res, name, 0, 0);
+        }
+    })
+    
+})
+
+app.get('/automated/source/:name/comoEsta', (req, res) => {
+    const {name} = req.params
+    const lastTilt = new Date().getTime();
+    
+    Tiltados.findOne({name}).then(savedTiltado => {
+        let record = 0;
+        if (savedTiltado) {
+            const daysDiff = daysDifference(new Date().getTime(), savedTiltado.lastTilt);
+            if (daysDiff > record) {
+                savedTiltado.record = daysDiff;
+            }
+            renderLastTilt(res, savedTiltado.name, daysDiff, savedTiltado.record < daysDiff ? daysDiff : savedTiltado.record);
+        } else {
+            res.send("usuário Inválido")
+        }
+    })
+    
+})
+
+
+app.get('/source/:name', (req, res) => {
```

Comment:
Why did you create this route? It doesn't have all the required parameters to render the correct image and it doesn't query the database to get the user's data, so I'm a little bit lost about it...

Reply:
Just an error during the copy and paste, the route was already removed.